data/Sri Dasam Granth/1389.json

```diff
@@ -247,7 +247,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "",
+              "translation": "Without any wealth, falcon, army property and authority,",
```

Comment:
Remove comma?

Reply:
I think this one might be missing the Oxford comma

Reply:
Ditto first point.

Reply:
Oxford comma is required, `falcon, army property, and authority` or `falcon, army, property, and authority`